{"name": "test_user_failure", "status": "passed", "start": 1746189682861, "stop": 1746189684656, "uuid": "f5aacbca-95e5-4773-9e56-50814293ae53", "historyId": "dd9ef04c5d3d39f51e7dadbc0a04546a", "testCaseId": "dd9ef04c5d3d39f51e7dadbc0a04546a", "fullName": "projekt_zaliczeniowy.tests.api.test_user#test_user_failure", "labels": [{"name": "feature", "value": "API"}, {"name": "story", "value": "Błędne pobranie informacji czy uzytkownik jest zalogowany"}, {"name": "tag", "value": "regression"}, {"name": "tag", "value": "api"}, {"name": "parentSuite", "value": "projekt_zaliczeniowy.tests.api"}, {"name": "suite", "value": "test_user"}, {"name": "host", "value": "DESKTOP-4S3FS01"}, {"name": "thread", "value": "24476-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "projekt_zaliczeniowy.tests.api.test_user"}]}